lib/Indexer/Adapter/ReferenceFinder/IndexedReferenceFinder.php

```diff
@@ -227,8 +227,9 @@ private function newObjectReferences(string $containerType): Generator
                     continue;
                 }
                 yield LocationConfidence::surely(
-                    Location::fromPathAndOffset(
+                    Location::fromPathAndOffsets(
                         $file->filePath() ?? '',
+                        $fileReference->offset(),
```

Comment:
`fromPathAndOffset`. does the indexer not support ranges?

Reply:
Range has been added